README.md

```diff
@@ -232,9 +230,9 @@ To customise existing block templates such as `Content` and `Form` templates, co
 `vendor/dnadesign/silverstripe-elemental/templates` to your theme. When doing this, ensure you match the folder
 structure (PHP class namespace) to ensure that your new template version takes priority.
 
-**Note:** The default set of elements follow the [BEM (Block Element Modifier])(http://getbem.com/) class naming
-convention, which allows developers to style individual parts of the DOM without unnecessarily nested CSS. Where
-possible, we encourage you to follow this naming system.
+> [!NOTE]
+> The default set of elements follow the [BEM (Block Element Modifier])(http://getbem.com/) class naming convention, which allows developers to style individual parts of the DOM without unnecessarily nested CSS.
+> Where possible, we encourage you to follow this naming system.
```

Comment:
Can you please revert this formatting change? Ideally we don't want any github-specific markdown, as that reduces the portability of the documentation and it is very likely that this documentation will be ported to the main developer-docs repo in the future.

Same with any other similar changes in this PR.

Reply:
Ah, k..no worries